config/m4/sysdep.m4

```diff
@@ -104,9 +104,7 @@ AC_CHECK_HEADERS([sys/platform/ppc.h])
 #
 # PowerPC query for getting TB frequency
 #
-AC_CHECK_DECL([__ppc_get_timebase_freq],
-              [AC_CHECK_FUNCS([__ppc_get_timebase_freq])], [],
-              [#include <sys/platform/ppc.h>])
+AC_CHECK_FUNCS([__ppc_get_timebase_freq])
```

Comment:
if I'm not mistaken it is worth checking for declaration and then for function itself

Reply:
I've got the following problems:
https://github.com/openucx/ucx/pull/7619#discussion_r743065020
https://github.com/openucx/ucx/pull/7619#discussion_r743075284

Reply:
AC_CHECK_FUNCS checks for function in standard libraries, we definitely have to include system headers to source file to use function.
as I can see ```sys/platform/ppc.h``` headed is part of libc package and must correspond to library

Reply:
sorry, this is a change for `__ppc_get_timebase_freq`, not for `__ppc_get_timebase`

Reply:
oh, I see - __ppc_get_timebase is declared as inline, but __ppc_get_timebase_freq - is libc call

Reply:
I bit more cleaned configs.
now it checks more natively :)